clients/src/main/java/org/apache/kafka/common/network/Selector.java

```diff
@@ -15,11 +15,7 @@
 import java.io.IOException;
 import java.net.InetSocketAddress;
 import java.net.Socket;
-import java.nio.channels.CancelledKeyException;
-import java.nio.channels.ClosedChannelException;
-import java.nio.channels.SelectionKey;
-import java.nio.channels.SocketChannel;
-import java.nio.channels.UnresolvedAddressException;
+import java.nio.channels.*;
```

Comment:
I don't think we want * imports...checkstyle should catch this on a clean build.